packages/api/src/api/videos.ts

```diff
@@ -0,0 +1,33 @@
+import {Router} from 'express';
+import { Video } from '../entities/Video';
+import logger from '../logger';
+
+export const videos = Router();
+
+// Get all videos
+videos.get('', async (req, res) => {
+    try {
+        const videosRepository = req.entityManager.getRepository(Video);
+        const videosList = await videosRepository.findAll();
+
+        res.status(200).send(videosList);
+    } catch (error) {
+        logger.error('There was an issue geting all videos: ', error);
+        res.status(500).send('There was an issue geting all videos');
+    };
+});
+
+// Get video by id
+videos.get('/:videoId', async (req, res) => {
+    const { videoId } = req.params;
+
+    try {      
+        const videosRepository = req.entityManager.getRepository(Video);
+        const video = await videosRepository.findOne({ id: videoId });
```

Comment:
We need to do a type check on `videoId`. If we don't the DB may throw an error if we give it a bad type and then we'll report a `500` back to the user when it should have been a `4xx` error.

Reply:
Added type checking and returning a 422 (Unprocessable Entity) if its not what we expect.

Reply:
Updated.

Reply:
I'll also keep looking into the mocking of the DBresponse.  Now that I know I also need to mock the entityManager I have more of an idea.